src/js/plugins/plugin.search.js

```diff
@@ -611,11 +622,13 @@ BookReader.prototype._searchPluginGoToResult = async function (pageIndex) {
 /**
  * Removes all search pins
  */
-BookReader.prototype.removeSearchResults = function() {
+BookReader.prototype.removeSearchResults = function(suppressFragmentChange = false) {
   this.removeSearchHilites(); //be sure to set all box.divs to null
   this.searchTerm = null;
   this.searchResults = null;
-  this.trigger(BookReader.eventNames.fragmentChange);
+  if (!suppressFragmentChange) {
+    this.trigger(BookReader.eventNames.fragmentChange);
```

Comment:
so, unless we flip it true, this will always fire, ya?

Reply:
Yes.